test/controllers/psp/PspSchemeDashboardControllerSpec.scala

```diff
@@ -261,16 +263,18 @@ class PspSchemeDashboardControllerSpec
       redirectLocation(result) mustBe Some(controllers.routes.ContactHMRCController.onPageLoad().url)
     }
 
+    /* TODO: No longer relevant after introduction of PspSchemeAuthAction PODS-8442. Refactor this eventually. -Pavel Vjalicin
```

Comment:
If we dont need this, can we delete the comment?

Reply:
Done